src/robot.coffee

```diff
@@ -288,7 +288,13 @@ class Robot
 
     app.use express.basicAuth user, pass if user and pass
     app.use express.query()
-    app.use express.bodyParser()
+
+    # Removing bodyParser since it's deprecated
+    # Note that Hubot no longer supports multipart uploads due to no longer using express.bodyParser
+    # In order to put support back for that, we would need to use one of: formidable, multiparty, busboy, multer, etc.
+    app.use express.urlencoded()
```

Comment:
Reading over http://www.senchalabs.org/connect/bodyParser.html, it says it's equivilant to:

```javascript
app.use(connect.json());
app.use(connect.urlencoded());
app.use(connect.multipart());
```

That makes me think the order is significant, so these should probably be swapped.